src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -8007,8 +7938,9 @@ components:
         type: string
         enum:
           - steward
-          - reader
```

Comment:
I think we still want reader and discoverer?

Reply:
Yes, you're right!